example/src/App.tsx

```diff
@@ -84,6 +86,11 @@ export default function App() {
     console.log('in verify');
     if (adaptor.localStream.current && adaptor.localStream.current.toURL()) {
       console.log('in verify if adaptor local stream', adaptor.localStream);
+      if (isWaitingWebsocketInit) {
+        setIsWaitingWebsocketInit(false);
+        publishStreamId = generateRandomString(12);
+        adaptor.publish(streamNameRef.current);
```

Comment:
Can this not be replaced with `handlePublish()` ?

Reply:
Yes, it's possible. You can make the publish stream id persistant.